.swiftlint.yml

```diff
@@ -1,3 +1,43 @@
+custom_rules:
+  missing_swifterswift_prefix:
+    included: ".*\\.swift"
+    excluded: ".*Tests\\.swift"
+    name: "Missing SwifterSwift: prefix"
+    regex: "///\\s(?!SwifterSwift:\\s)\\w+[^\\n]*$"
+    match_kinds:
+      - doccomment
+    message: "Doc string should always start with the SwifterSwift: prefix"
+    severity: warning
+  unnecessary_type:
+    name: "Unnecessary Type"
+    regex: '(?sm)[ \ta-zA-Z0-9]?(?:let|var){1} [ \ta-zA-Z0-9]+?:[ \t]+?([a-zA-Z0-9]+?)[\t ]+?=[\t ]?\1'
+    message: "Type Definition Not Needed"
+    severity: warning
+  comments_space:
+    name: "Space After Comment"
+    regex: '(//\w+|///\w+)'
+    message: "There should be a space between the start of a comment and the first character."
+    severity: warning
+    match_kinds:
+      - comment
+      - doccomment
+      - doccomment.field
+  empty_commented_line:
+    name: "Empty Commented Out Line"
+    regex: '(\t+| +)//\n'
+    message: "Remove useless comment lines or use /* format */"
+    severity: warning
+  invalid_mark_format:
```

Comment:
Does this one already exists I think - [Mark](https://github.com/realm/SwiftLint/blob/2dcaf3ee7803e25c8163e2fbbb9b8ac2be722a2d/Rules.md#L11971)

Reply:
Yep, will remove it